src/main/resources/templates/sales/index.html

```diff
@@ -19,7 +19,7 @@
 </head>
 
 <body>
-<nav th:replace="sales/fragments/header :: header"></nav>
+<nav th:replace="/sales/fragments/header :: header"></nav>
```

Comment:
I don't think this / should have been added.

Reply:
Fixed!